src/psalm-language-server.php

```diff
@@ -146,11 +148,22 @@ function ($arg) use ($valid_long_options, $valid_short_options) {
 
     --enable-autocomplete[=BOOL]
         Enables or disables autocomplete on methods and properties. Default is true.
+
+    --use-extended-diagnostic-codes
+        Enables sending help uri links with the code in diagnostic messages.
+
+    --verbose
+        Will send log messages to the client with information.
 HELP;
 
     exit;
 }
 
+if (array_key_exists('v', $options)) {
+    echo 'Psalm ' . PSALM_VERSION . PHP_EOL;
+    exit;
+}
+
```

Comment:
moved this up to allow for an early exit since they just want the version.  There is no need to check other stuff before this.